### DISCUSSION TITLE
I want to play again

### DISCUSSION THREAD

**Discussion by czizsu** [stepmania/stepmania#2260] [Q&A]
Hello, I have a StepMania carpet, and I want to play again after 10 years. Is there any game which is working (Windows)? I installed a version but it is not running. Thanks

**Comment by reynerth** [comment]
You might need to setup the default stepmania build adding songs themes etc, but an option is to find over the internet some ready to play custom-builds made by the community, also in case you're sure you have a correctly setup build but the game doesnt starts, try checking the Antivirus or something that might be preventing the normal execution of the game, i use win10 and works fine for me, so i can confirm the lastest build of stepmania shouldnt have any compatibility problem, i might try to help you if you provide some more info about the problem...
Hope this help..
